util/Migrator/DbScripts/2021-07-01_00_EntityFrameworkSupport.sql

```diff
@@ -0,0 +1,1047 @@
+-- Tech debt: creates a few views and procedures that we implement as base repository methods but that would throw exceptions on use for not existing in the DB
```

Comment:
Okay, so please rename this file again with a more current date so it post-dates the latest release. Say Jul 8.